public/locales/en/translation.json

```diff
@@ -338,7 +343,6 @@
     "Angola": "Angola",
     "Anguilla": "Anguilla",
     "Anniversary": "Anniversary",
-    "Annual": "Annual",
```

Comment:
Notice that the old version of the i18next extractor considers those four labels to be no longer also.

Reply:
Replied in this PR: https://github.com/CruGlobal/mpdx-react/pull/1188#pullrequestreview-2428181996